source/utils/types.ts

```diff
@@ -1,14 +1,19 @@
 import {IncomingMessage} from 'http';
 import {RequestOptions} from 'https';
+import {Readable} from 'stream';
```

Comment:
`import {Readable as ReadableStream} from 'stream';` to improve code readability.